java/core/src/main/java/co/worklytics/psoxy/storage/StorageHandler.java

```diff
@@ -35,7 +36,7 @@ public class StorageHandler {
     Pseudonymizer pseudonymizer;
 
     @Inject
-    BulkDataRules defaultRuleSet;
+    RuleSet defaultRuleSet;
```

Comment:
this change REQUIRED before `MultiTypeBulkDataRules` will actually work in practice, bc `MutliTypeBulkDataRules` don't implement `BulkDataRules`  (they could, but that's also a code change ...